extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -130,15 +135,14 @@ var CreatePage = {
 					}
 				};
 				uiModal.createComponent( createPageModalConfig, function( createPageModal ) {
-					CreatePage.track( { action: 'impression', label: 've-redlink-modal' } );
+					CreatePage.track( { action: Wikia.Tracker.ACTIONS.IMPRESSION, label: 'modal' } );
 
-					createPageModal.bind( 'create', function( event ) {
-						CreatePage.track( { action: 'click', label: 've-redlink-create' } );
```

Comment:
It's added `submitDialog()` method: https://github.com/Wikia/app/pull/11923/files#diff-83762097831c15f251e93d8c8fd39d5cR264